lib/scarpe/button.rb

```diff
@@ -17,11 +21,24 @@ def click(&block)
 
     def render
       @app.bind(function_name) do
-        if @block
-          @block.call
-        end
+        @block.call if @block
       end
-      "<button id=#{object_id} onclick='scarpeHandler(#{function_name})'>#{@text}</button>"
+      "<button id=#{object_id} onclick='scarpeHandler(#{function_name})' style='#{style}'>#{@text}</button>"
```

Comment:
```suggestion
      puts "<button id=#{object_id} onclick='scarpeHandler(#{function_name})' style='#{style}'>#{@text}</button>"
      "<button id=#{object_id} onclick='scarpeHandler(#{function_name})' style='#{style}'>#{@text}</button>"
```

I'd like to keep up Marco's pattern of logging the resulting HTML to the terminal, is that okay? 

I know it looks a bit funky 'puts'ing something and then returning it. But right now I don't see a sneaky way around it. Unless we create a parent private method `render` with a `super` or something to do a bit of cleverness. But I think we can be repetitive for now?

Reply:
maybe just assign it to a variable? ;)

Reply:
Noah had a good point about debug functionality going forward, which may get more elegant, so I've merged! :D

Reply:
aye it'd be a shorter

```ruby
puts @some_stuff
@some_stuff
```

but I think we'll do better